### PR TITLE
Add tomli to library requirements

### PR DESCRIPTION
**Pull request by habospace**
Adding tomli to requirements since it isn't a default package in all python environments

### CODE REVIEW

**Comment by olliemath** [comment]
Can you make the requirement conditional on python version < 3.11 - like here: https://stackoverflow.com/questions/19559247/requirements-txt-depending-on-python-version

And then for the import in >= 3.11 you can use stdlib:
```python
try:
    import tomllib as tomli  # python >= 3.11
except ImportError:
    import tomli  # python < 3.11
```

File: rattr/cli/toml_parser.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
 from pathlib import Path
 from typing import Any, Dict, Optional
 
-import tomli as tomllib
+try:
+    # python >= 3.11
+    import tomllib
+except ImportError:
+    # python < 3.11
+    import tomli as tomlib  # noqa: F401
```
Comment (olliemath) [inline rattr/cli/toml_parser.py:9]: The F401 is legit - there's a spelling error here `tomlib` vs `tomllib`

Comment (olliemath) [reply rattr/cli/toml_parser.py:9]: @habospace in case you hadn't seen this

Comment (habospace) [reply rattr/cli/toml_parser.py:9]: I did. I will update this afternoon.

**Review by olliemath — APPROVED** [review]
LGTM

**Pull request merged**